### ISSUE TITLE
Can't move message content with cursor keys/pgup/pgdn/home/end

### ISSUE THREAD

**Issue by Steltek** [RainLoop/rainloop-webmail#1574]
Rainloop 1.11.3, browsers Chrome/Firefox, OS Windows 10/Debian Linux

After clicking on an email/into an email body, using the cursor keys (or other nav keys like PgDn, ...) should move the message body/contents. Instead, these keys move the message cursor (switching to different messages when pressed), leaving the user with no way of reading the message using the keyboard. (The only way to see the next page is to use the mouse cursor to scroll down.)

Use Vertical-split layout, open multi-page email, try to get to 2nd page of email using keyboard.

**Comment by megadr01d** [comment]
Click on the main menu (next to your email) and choose **Help > Message list**.
You need to use <kbd>Right/Left Cursor</kbd> or <kbd>Tab</kbd> to switch from the message list to the message preview.

**Comment by Steltek** [comment]
IMHO Having to use the keyboard to switch focus is counter-intuitive to UX standards established by other software out there (Windows Explorer, Outlook, Thunderbird, Roundcube, ...). The universally used UX standard is that if you click into a window, or any part with a scrollbar, it receives focus and navigation keys apply to it. (If I click outside of the window I'm typing this in, that's what gets moved by my cursor keys. If I click into the text box, the cursor and PgUp/PgDn/Home/End keys apply to it. 

Being able to use keys to switch focus between Message List and Preview is a nice addition, but it should not be the only way to do it. /IMHO

**Comment by megadr01d** [comment]
> Being able to use keys to switch focus between Message List and Preview is a nice addition, but it should not be the only way to do it.

You're right. I thought you were trying to achieve it only by using shortcut keys.
